tests/dummy/app/pods/select/template.hbs

```diff
@@ -1,5 +1,4 @@
 <div class="dummy-body select">
-  <div class="section-header">frost-select</div>
```

Comment:
Already done in the last commit

From: Gosia Hyndman <notifications@github.com<mailto:notifications@github.com>>
Reply-To: ciena-frost/ember-frost-core <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, April 5, 2016 at 11:44 AM
To: ciena-frost/ember-frost-core <ember-frost-core@noreply.github.com<mailto:ember-frost-core@noreply.github.com>>
Cc: Panthaky Roxanne <rpanthak@ciena.com<mailto:rpanthak@ciena.com>>
Subject: Re: [ciena-frost/ember-frost-core] updated multi select checkbox size. fix margin issue for some demos. (#40)


In tests/dummy/app/pods/select/template.hbs<https://github.com/ciena-frost/ember-frost-core/pull/40#discussion_r58562490>:

> @@ -1,5 +1,4 @@
>  <div class="dummy-body select">
> -  <div class="section-header">frost-select</div>


So this one needs to came back.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ciena-frost/ember-frost-core/pull/40/files/55fbdef80c1d9d446ff14aa6ccb05794033aa59d..c5db8aae06ca9e94f93ba52673abf96b5afa32c0#r58562490>